libcaf_core/caf/detail/stream_distribution_tree.hpp

```diff
@@ -229,6 +230,7 @@ class stream_distribution_tree : public stream_manager {
       return policy_.path_force_dropped(sid, hdl, std::move(reason));
     return none;
   }
+  */
```

Comment:
Happens in clang based builds [in Jenkins](https://jenkins.inet.haw-hamburg.de/job/CAF/job/develop%20branch%20clang/label_exp=Linux%20&&%20clang%20&&%20LeakSanitizer/680/console) as well.